Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift

```diff
@@ -192,11 +192,15 @@ internal extension UIView {
             return subview.asViewElement(in: bounds, safeAreaInsets: childInsets, autoTag: childAutoTag)
         }
 
-        // only create a selector for elements that have at least the center point 
-        // visible in the current screen bounds, inset by any safe area adjustments
+        // find the rect of the visible area of the view within the safe area
         let safeBounds = bounds.inset(by: safeAreaInsets)
-        let centerPointVisible = safeBounds.contains(CGPoint(x: absolutePosition.midX, y: absolutePosition.midY))
         let visibleRect = safeBounds.intersection(absolutePosition)
+        // if there is no intersection of this view in the safe area, it is skipped
+        guard !visibleRect.isNull else { return nil }
```

Comment:
only question I have is if this opens some edge case where a parent is not in the safe area but it's children are (outside its bounds)?

Reply:
reworked this - now it will not skip the element, so any children are still potentially captured in selectors. There will not be a selector generated for the non-visible element, since it's center point is not in the safe bounds (L207 check)